profile/README.md

```diff
@@ -0,0 +1,3 @@
+![Polypass](https://raw.githubusercontent.com/polypass/polypass-branding/main/SVG/POLYPASS_RGB_FullColor_Inverted.svg)
```

Comment:
Can you constrain the size of this to a reasonable size? Leave the width alone, just constrain the height.